### ISSUE TITLE
Module 11 - Task 03 - Notebook 3 - Lab Setup

### ISSUE THREAD

**Issue by bcm-agahgnango** [ctesta-oneillmsft/xyz#36]
# Module: 11
#### Step: Exercise 03

Description of issue

Lab setup section of the 3.Streaming-With-Event-Hubs-Demo notebook of that exercise 3  is hard to follow being in the Databricks workspace. Installing the library takes about 7 steps to complete. The issue here is that when learners start the install process, they lose sight of the instructions that are in the notebook. As it stands right now, they will have to memorize this specific set of instructions before they start.

Repro steps:

1.
1.
1.

Alternative : Please move this Lap Setup section in between the Exercise 1- task 1 and Exercise 1- task 2.

**Comment by ctesta-oneillmsft** [comment]
Its a good opportunity for the students to remember how to get back to a notebook when they move away from it, and is by design
